documentation/staging/content/release-notes.md

```diff
@@ -6,52 +6,104 @@ draft: false
+* Cluster resource: the operator now provides a new custom resource, Cluster, which configures a specific WebLogic cluster. This resource can be used with Kubernetes Horizontal Pod Autoscaler (HPA) or similar technologies to scale WebLogic clusters.
+* Status and Events: the status of and the Events generated about Domain and Cluster resources have been significantly updated. The operator more clearly communicates when the domain or individual clusters have reached the intended state or significant availability for your application workloads. Failures are distinguished between those that require customer intervention or those that are potentially temporary and will be retried.
+* Istio-compatibility: domain configurations are always compatible with Istio and other service mesh products. Customers are no longer required to enable this support.
+
+##### Upgrade Notes
+
+* This release introduces a new API version for the Domain resource, `weblogic.oracle/v9`. The previous API version, `weblogic.oracle/v8` introduced with Operator 3.0.0 is deprecated, but is still supported.
+* The operator provides a [schema conversion webhook]({{< relref "/managing-operators/conversion-webhook.md" >}}) that automates the upgrade of `weblogic.oracle/v8` Domain resources to `weblogic.oracle/v9`. Customers do not need to take any action to begin using the latest Domain schema; however, the team recommends starting with "v9" Domain resources for any new projects. All samples, tools, and documentation have been updated to use "v9". The operator provides a library that can be used to update existing "v8" Domain YAML files to "v9".
+* This release also introduces a new custom resource, the `weblogic.oracle/v1` Cluster resource. This Cluster resource configures a specific WebLogic cluster and allows customers to scale that cluster using the Kubernetes Horizontal Pod Autoscaler (HPA) or similar technologies. The Cluster resource schema is similar to the former `weblogic.oracle/v8` Domain resource `.spec.clusters[*]` content.
+* Customers who install a single WebLogic Kubernetes Operator per Kubernetes cluster (most common use case) can upgrade directly from any 3.x operator release to 4.0.0. The Helm chart for 4.0.0 will automatically install the schema conversion webhook.
+* For those customers who install more than one WebLogic Kubernetes Operator in a single Kubernetes cluster, you must upgrade every operator to at least version 3.4.1 before upgrading any operator to 4.0.0. As the 4.0.0 Helm chart now also installs a schema conversion webhook, you will want to use the Helm chart option to install this webhook in its own namespace prior to installing any of the 4.0.0 operators.
+* Several Helm chart default values have been changed. Customers who upgrade their 3.x installations using the `--reuse-values` option during the Helm upgrade will continue to use the values from their original installation.
```

Comment:
- Perhaps call this an 'upgrade checklist' and make this more of a cookbook style set of steps?
- Cluster resource bullet: If I understand correctly, not only do we introduce a new Custom Cluster Resource, but also a Cluster Resource is automatically created as needed, or upon upgrade. Should this automation be mentioned in the upgrade section? In addition, I assume that any scripts/automation that depend on the old 'replicas' or other life-cycle settings location for a Cluster in the old location will stop working. Should that be mentioned?
- Potential new bullet: Do we need to tell users that currently install their domain CR using a custom approach that they'll also need to do the same for the cluster CR?
- In the bullet where the changes to helm defaults are mentioned, consider linking or mentioning the enumeration of changed default values below.
- Potential new bullet: Add a mention condition and event changes in the upgrade notes? Or are we assuming that no current customer is depending on the old condition/event behavior?
- All bullets: Consider adding TBD or working links for related doc for all MajorTheme/Upgrade items that have it or will have it (I assume conditions is going to have doc, events already has new doc)
- Potential new bullet:  Mention the new retry behavior/tuning in 'upgrade' or 'major themes'?
- About "For those customers who instell more than one WKO...". One additional concern is that they'll probably want to independently install the webhook. Otherwise a de-install of one of the operators will unexpectedly take down all access to the domain resource...

Reply:
As I mentioned to @rosemarymarano, I'd created my release note sections by looking at what K8s does for its own release notes. They have a much more extreme sounding title: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md#urgent-upgrade-notes "Urgent Upgrade Notes" and "No, really, you MUST read this before you upgrade".

Reply:
BTW, from a technical writing POV, I would not suggest following the lead of the Kubernetes docs. Just sayin'.

Reply:
Ha! Sure. I needed a starting point and knew that I would have a review. :)

Reply:
"No, really, you MUST read this before you upgrade" 

LOL - The major version change is already a pretty big hint